lib/client/fast_render.js

```diff
@@ -18,10 +18,17 @@ if(FastRender._disable) {
 //  This is the cure
 FastRender.injectDdpMessage = function(conn, message) {
   FastRender["debugger"].log('injecting ddp message:', message);
-  var originalWait = conn._waitingForQuiescence;
-  conn._waitingForQuiescence = function() {return false};
-  conn._livedata_data(message);
-  conn._waitingForQuiescence = originalWait;
+  if (conn._bufferedWrites) {
```

Comment:
Not sure why I had things working at some point, but indeed, that makes sense.   Looks like @dnish already got the fix applied for this.  Good catch!